corehq/apps/app_manager/suite_xml/case_tile_templates/tdh.txt

```diff
@@ -58,12 +58,17 @@
       </header>
       <template>
         <text>
-          &lt;b&gt;{header[prefix]} &lt;/b&gt; <xpath function="{header[xpath_function]}"/>
+          &lt;b&gt;{header[prefix]} &lt;/b&gt;
+          <xpath function="{header[xpath_function]}">
+            {header[variables]}
```

Comment:
Ah, that makes lots of sense. Looking again now, I had misinterpreted this as a _test_ file, but it's actually a production file for the custom case tiles "feature" we wrote for tdh.

Thanks for the explanation.